webapp/pages/admin/users.vue

```diff
@@ -128,6 +134,7 @@ export default {
               id
               name
               slug
+              email
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-03T23:46:33Z" title="Friday, October 4th 2019, 1:46:33 am +02:00">Oct 4, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -128,6 +138,7 @@ export default {
               id
               name
               slug
+              email
```


At the moment, the backend does not allow anybody except the owner to see the `email` of the user. You will need to change permissions in the backend.

I think I would still feel safe, if only the owner *or* an admin user can see the email.

Reply:
> [<img alt="aonomike" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/aonomike) **Authored by [aonomike](https://github.com/aonomike)**
_<time datetime="2019-10-05T14:24:29Z" title="Saturday, October 5th 2019, 4:24:29 pm +02:00">Oct 5, 2019</time>_

---


Done, thanks @roschaefer